rules/book-developers-for-a-project/rule.md

```diff
@@ -88,9 +87,9 @@ Sometimes you may want to reserve a consultant for an appointment but are not ab
 To create a Tentative Booking:
 
 1. Follow the same steps above to create the Appointment
-2. Instead of choosing a client account for 'Set Regarding', choose the company (E.g. "ssw.pencilledin").
+2. Instead of choosing a client account for 'Set Regarding', choose the company **ssw.pencilledin**.
```

Comment:
Hey @samsmithssw 

As per Adam's email: **SSW Rules should be generic**

1. Make this sentence:

> 2. Instead of choosing a client account (e.g. Northwind) for 'Set Regarding', choose a pre-defined "company" account for tentatives (e.g. SSW.PencilledIn).

Reply:
Hey @tiagov8 

I don't think that rule works in this case as it's specifically talking about how you use the pencilled.in function. So there is no need to be generic in this case.